src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/activation.cpp

```diff
@@ -95,6 +95,15 @@ void ActivationLayerCPUTest::generate_inputs(const std::vector<ov::Shape>& targe
             // cover Sign NAN test case
             if ((activationType == utils::ActivationTypes::Sign) && funcInput.get_element_type() == ov::element::f32) {
                 static_cast<float*>(tensor.data())[0] = std::numeric_limits<float>::quiet_NaN();
+            } else if ((activationType == utils::ActivationTypes::IsFinite) && funcInput.get_element_type() == ov::element::f32 && tensor.get_size() >= 8) {
+                static_cast<float*>(tensor.data())[0] = std::numeric_limits<float>::quiet_NaN(); // nan
+                static_cast<float*>(tensor.data())[1] = std::numeric_limits<float>::signaling_NaN(); // nan
+                static_cast<float*>(tensor.data())[2] = std::acos(2); // nan
+                static_cast<float*>(tensor.data())[3] = std::tgamma(-1); // nan
+                static_cast<float*>(tensor.data())[4] = std::log(-1); // nan
+                static_cast<float*>(tensor.data())[5] = std::sqrt(-1); // -nan
+                static_cast<float*>(tensor.data())[6] = 0 / 0.0; // -nan
```

Comment:
@awayzjj This line leads to failed Win build on CI: 
"openvino/src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/activation.cpp(105): error C2124: divide or mod by zero"

Reply:
@dmitry-gorokhov Thank you! But I do not have much experirence on windows. Should I just remove the line 105?

Reply:
You are implementing isFinite operation. So I believe it is enough to have 4 specific values:  nan, -nan, inf, -inf

Reply:
Thank you, I will fix it later today!